src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
Mmh, if I use ``E_SOMETHING_UNRELATED``, I would like see it in the dump.

More, if I define correclty a new E_ constant like ``define('E_CUSTOM', 32768)``, it will not be dumped in your case.